sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -59,6 +60,17 @@
 if TYPE_CHECKING:
     import datetime
 
+PrimitiveTypes = Union[
+    None,
```

Comment:
Should this be NoneType? Or maybe instead the whole union should be wrapped in Optional?